api/src/main/java/org/ccci/gto/globalreg/ServerErrorException.java

```diff
@@ -3,19 +3,10 @@
 /**
  * Created by ryancarlson on 8/29/14.
  */
-public class ServerErrorException extends GlobalRegistryException {
-	final int statusCode;
-	final String responseContent;
+public class ServerErrorException extends HttpErrorException {
 
 	public ServerErrorException(int statusCode, String responseContent) {
-		this.statusCode = statusCode;
-		this.responseContent = responseContent;
-	}
-
-	public ServerErrorException(Throwable cause, int statusCode, String responseContent) {
-		super(cause);
-		this.statusCode = statusCode;
-		this.responseContent = responseContent;
+		super(statusCode, responseContent);
 	}
 
 	public int getStatusCode() {
```

Comment:
I believe you can get rid of `getStatusCode` and `getResponseContent` here, they should be available from the parent class.

Reply:
indeed, thanks